envi/archs/arm/disasm.py

```diff
@@ -892,7 +923,7 @@ def p_media_pack_sat_rev_extend(opval, va):
             
     elif (opc1 & 2) and opc25 == 1: #word sat
         opidx = (opval>>22)&1
-        sat_imm = 1 + (opval>>16) & 0xf
+        sat_imm = (opval>>16) & 0xf
```

Comment:
Backed by reference manual, both old and new. Looks like was an assumption was made... Also checked with online disassembler and assembler (Different units).